src/pages/Clients/Eth1/Besu.tsx

```diff
@@ -73,24 +73,28 @@ export const BesuDetails = () => (
           </Link>
         </>
       )}
-      <Text className="mt10">
-        <FormattedMessage
-          defaultMessage="Use {http} to connect your Eth2 node to the JSON RPC
-          endpoint. This will enable the JSON RPC services on the default 8545
-          port."
-          values={{
-            http: <Code className="mt20">-rpc-http-enabled</Code>,
-          }}
-          description="{http} shows '-rpc-http-enabled' terminal command"
-        />
-      </Text>
-      <Link
-        primary
-        to="https://besu.hyperledger.org/en/stable/Reference/CLI/CLI-Syntax/#rpc-http-enabled"
-        className="mt20"
-      >
-        <FormattedMessage defaultMessage="rpc-http-enabled documentation" />
-      </Link>
+      {IS_MAINNET && (
```

Comment:
Are you sure this should only be shown if on mainnet?

I think the point was that it appears like it's under the "Testing on Goerli" header when that section is present:
![Screen Shot 2021-02-25 at 6 09 03 PM](https://user-images.githubusercontent.com/8097623/109245019-98f52580-7794-11eb-8200-92f253fe167c.png)

Perhaps @hwwhww can clarify their comment on https://github.com/ethereum/eth2.0-deposit/pull/308#discussion_r582951631

Reply:
Ah sorry, I didn't clarify enough.
I meant the json-rpc setting could be set for both mainnet or goerli.

So IMO the headers could be either (1):

### Configurations
Use `--config goerli` ....
Use `--JsonRPC.Enabled true`...

---
Or (2):

### Testing on Goerli
Use `--config goerli` ....
### JSON RPC endpoint
Use `--JsonRPC.Enabled true`...

Reply:
Gotcha! Should've clarified, patching this up now